### PR TITLE
Pin ccache-action to a version that updates the PATH

### PR DESCRIPTION
**Pull request by kendalharland**
See https://github.com/hendrikmuhs/ccache-action/pull/204. This is necessary for this workflow to run on custom runners where `$USERPROFILE/.cargo/bin/` or any future installation path isn't already part of $GITHUB_PATH

### CODE REVIEW

**Review by tristanlabelle — APPROVED** [review]

**Comment by compnerd** [comment]
I think that we should wait a bit and see if upstream responds. IIRC, they were pretty responsive, so giving them 24-48h seems reasonable.

**Comment by kendalharland** [comment]
> I think that we should wait a bit and see if upstream responds. IIRC, they were pretty responsive, so giving them 24-48h seems reasonable.

I think this is reasonable. At the same time, waiting for upstream reviews from someone I can't directly contact, who also has their own priorities outside the project creates an undesirable slow down :/ I wonder if there is a less distributed strategy for managing changes to the 3p actions we depend on, such as keeping our own forks that are auto-synced, and retroactively upstreaming all changes when possible.

**Comment by kendalharland** [comment]
@compnerd let's merge! Seems like upstream is not attending the PR at the moment

**Comment by kendalharland** [comment]
Upstream has merged the change and this is no longer needed. I'll send a separate PR to revert back to hendrikmuhs/ccache-action

**Pull request closed without merging**
